envergo/hedges/static/hedge_input/app.js

```diff
@@ -382,6 +420,11 @@ createApp({
         // We don't restore ids, but since we restore hedges in the same order
         // they were created, they should get the correct ids anyway.
         const hedge = addHedge(type, latLngs, additionalData);
+        if(type === "TO_PLANT" && mode=== "removal") {
```

Comment:
Quelques petites incohérences de convention ici.

```suggestion
        if (type === "TO_PLANT" && mode === "removal") {
```